cocos/animation/marionette/animation-graph-context.ts

```diff
@@ -742,6 +776,30 @@ export class AnimationGraphUpdateContextGenerator {
 
 interface ReusableUpdateContext extends AnimationGraphUpdateContext {
     deltaTime: number;
-
     indicativeWeight: number;
 }
+
+export class DeferredPoseStashAllocator implements PoseStashAllocator {
```

Comment:
what's difference between deferred and not deferred?

Reply:
"Deferred" means the bone layout has not been determinate. Only after the bone layout has been determinate, the allocator can then be initialized.